kademlia/src/test/java/ingsw/group1/kademlia/NodesRoutingTableTest.java

```diff
@@ -0,0 +1,266 @@
+package ingsw.group1.kademlia;
```

Comment:
Concerning the testing methods, personally I would have preferred to use names such as "getLocationTest", "getClosestTest"... instead of "getLocation", "getClosest"...

Reply:
In my opinion it would be redundant to have "test" in the method signature since this is the test class.